iphone/templates/module/swift/hooks/install-xcode-templates.js

```diff
@@ -3,18 +3,28 @@
  * Hook that makes sure the Titanium Xcode file templates get installed.
  *
  * @copyright
- * Copyright (c) 2014-present by Appcelerator, Inc. All Rights Reserved.
+ * Copyright (c) 2018-present by Appcelerator, Inc. All Rights Reserved.
  *
  * @license
  * Licensed under the terms of the Apache Public License
  * Please see the LICENSE included with this distribution for details.
  */
 
+'use strict';
+
 var path = require('path');
-var fs = require('fs-extra');
 
 exports.cliVersion = '>=3.2';
 
+/**
+ * Main entry point for our plugin which looks for the platform specific
+ * plugin to invoke.
+ *
+ * @param {Object} logger The logger instance.
+ * @param {Object} config The hook config.
+ * @param {Object}cli The Titanium CLI instance.
+ * @param {Object} appc The Appcelerator CLI instance.
+ */
 exports.init = function (logger, config, cli, appc) {
 	cli.on('create.post.module', function (creator) {
```

Comment:
<!--
  0 failure: 
  1 warning:  iphone/templates/...
  
  
  DangerID: danger-id-default;
  File: iphone/templates/module/swift/hooks/install-xcode-templates.js;
  Line: 29;
-->

- :warning: iphone/templates/module/swift/hooks/install-xcode-templates.js line 29 – 'creator' is defined but never used. (no-unused-vars)